paasta_tools/paasta_metastatus.py

```diff
@@ -660,20 +707,9 @@ def main():
 
     healthy_exit = True if all([mesos_ok, marathon_ok, chronos_ok]) else False
 
-    if args.verbose == 0:
-        print mesos_summary
-        print marathon_summary
-        print chronos_summary
-    elif args.verbose == 1:
-        print mesos_summary
-        print_results_for_healthchecks(mesos_ok, all_mesos_results, args.verbose)
-        print marathon_summary
-        print_results_for_healthchecks(marathon_ok, marathon_results, args.verbose)
-        print chronos_summary
-        print_results_for_healthchecks(chronos_ok, chronos_results, args.verbose)
-    else:
-        print mesos_summary
-        print_results_for_healthchecks(mesos_ok, all_mesos_results, args.verbose)
+    print "Master paasta_tools version: {0}".format(__version__)
+    print_results_for_healthchecks(mesos_summary, mesos_ok, all_mesos_results, args.verbose)
+    if args.verbose > 1:
```

Comment:
Huh. Sure looks like we are losing something here, but I guess by printing the summary in the healchecks stuff, we don't need such verbose if/else logic?